.vib/ejbca/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+ejbcaCA:
+  name: ManagementTestCA
+  baseDN: "UID=c-1CCXmPAsNWmZuDtQQ8FHl7tcVdjCiNTH,O=ExampleCA,C=SE"
+podSecurityContext:
+  fsGroup: 1002
+containerSecurityContext:
+  runAsUser: 1002
+containerPorts:
+  http: 8081
+  https: 8443
```

Comment:
Have you encountered issues changing the default value for `containerPorts.https`? It seems strange the `http` one can be changed but not the other one.

Reply:
Yes, changes in the values for this port do not actually change the port in the application (unlike the `http` one)